testsupport/metrics/metrics.go

```diff
@@ -24,7 +24,7 @@ func GetMetricValue(restConfig *rest.Config, url string, family string, expected
 	client := http.Client{
 		Timeout: time.Duration(10 * time.Second),
 		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, //nolint:gosec
```

Comment:
just out of curiosity - what kind of error do we ignore here?

Reply:
`gosec` complains if we explicitly skip cert verification